packages/typo3-docs-theme/src/Inventory/Typo3InventoryRepository.php

```diff
@@ -2,22 +2,37 @@
 
 namespace T3Docs\Typo3DocsTheme\Inventory;
 
+use phpDocumentor\Guides\Interlink\DefaultInventoryLoader;
 use phpDocumentor\Guides\Interlink\Inventory;
-use phpDocumentor\Guides\Interlink\InventoryLoader;
 use phpDocumentor\Guides\Interlink\InventoryRepository;
+use phpDocumentor\Guides\Interlink\JsonLoader;
 use phpDocumentor\Guides\ReferenceResolvers\AnchorReducer;
+use Psr\Log\LoggerInterface;
 use RuntimeException;
+use Symfony\Component\HttpClient\Exception\ClientException;
 use T3Docs\Typo3DocsTheme\Settings\Typo3DocsThemeSettings;
 
 final class Typo3InventoryRepository implements InventoryRepository
 {
+    /**
+     * @see https://regex101.com/r/EXCPkt/1
+     *
+     * https://getcomposer.org/doc/04-schema.md#name
+     */
+    private const EXTENSION_INTERLINK_REGEX = '/^ext\/([^\/]*)\/(.*)$/';
```

Comment:
```suggestion
    private const EXTENSION_INTERLINK_REGEX = '/^ext\/([^\/]+)\/(.+)$/';
```

To avoid `ext//` which would be possible before.